### PR TITLE
Feature/elasticsearch logging

### PR DESCRIPTION
**Pull request by vigneswararaomacharla**
Review the Elasticsearch Logging issue enabled or not code changes.

### CODE REVIEW

File: hammer/library/aws/elasticsearch.py [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    @classmethod
+    def validate_access_policy(cls, policy_details):
+        """
+
+        :param policy_details: 
+        :return: 
+        """
+        public_policy = False
+        for statement in policy_details.get("Statement", []):
+            effect = statement['Effect']
```
Comment (oleksii-dowjones) [inline hammer/library/aws/elasticsearch.py:153]: Can we reuse this https://github.com/dowjones/hammer/blob/master/hammer/library/aws/s3.py#L69?

File: hammer/library/aws/elasticsearch.py [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+            elif node_to_node_encryption and node_to_node_encryption["Enabled"]:
+                domain_encrypted = True
+
+            logging_details = domain_detail.get("LogPublishingOptions")
+
+            if logging_details:
+                index_logs = logging_details.get("INDEX_SLOW_LOGS")
+                search_logs = logging_details.get("SEARCH_SLOW_LOGS")
+                error_logs = logging_details.get("ES_APPLICATION_LOGS")
+                if (index_logs and index_logs["Enabled"]) \
```
Comment (oleksii-dowjones) [inline hammer/library/aws/elasticsearch.py:347]: Should we enable only one or all three? Let's discuss.

File: hammer/reporting-remediation/reporting/create_elasticsearch_domain_logging_issue_tickets.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                        f"*Account Name*: {account_name}\n"
+                        f"*Account ID*: {account_id}\n"
+                        f"*Region*: {region}\n"
+                        f"*Domain ID*: {domain_name}\n"
+                    )
+
+                    issue_description += JiraOperations.build_tags_table(tags)
+
+                    auto_remediation_date = (self.config.now + self.config.esLogging.issue_retention_date).date()
+                    issue_description += f"\n{{color:red}}*Auto-Remediation Date*: {auto_remediation_date}{{color}}\n\n"
```
Comment (oleksii-dowjones) [inline hammer/reporting-remediation/reporting/create_elasticsearch_domain_logging_issue_tickets.py]: Do we need to add this comment if auto-remediation is actually disabled?

Comment (vigneswararaomacharla) [reply hammer/reporting-remediation/reporting/create_elasticsearch_domain_logging_issue_tickets.py]: Added condition based autoremediation.

File: hammer/library/aws/elasticsearch.py (hunk 2 above)
Comment (vigneswararaomacharla) [reply hammer/library/aws/elasticsearch.py:347]: Atleast one logging should be enable, so added for Error logs. But while identification we will check all three.

File: hammer/library/aws/elasticsearch.py (hunk 1 above)
Comment (vigneswararaomacharla) [reply hammer/library/aws/elasticsearch.py:153]: yeah, will reuse with adding in new file (policy.py) and will use for common both S3, IAM and ES (other use cases if any). 
Need to test all use cases. Will check with pranav and do code separation. For this, we can merge and deploy.

**Pull request merged**